mscore/newwizard.cpp

```diff
@@ -57,6 +57,7 @@ TimesigWizard::TimesigWizard(QWidget* parent)
       connect(tsCutTime,    SIGNAL(toggled(bool)), SLOT(cutTimeToggled(bool)));
       connect(tsFraction,   SIGNAL(toggled(bool)), SLOT(fractionToggled(bool)));
       pickupMeasure->setChecked(false); // checked in the UI file to enable screen reader on pickup duration controls
+      tempoGroup->setChecked(false);
```

Comment:
Might be worth a comment in the code?

Reply:
Even though it’s the same comment as the line above?

Reply:
Ah, missed that